src/core/lib/surface/server.c

```diff
@@ -123,7 +123,7 @@ typedef enum {
   ZOMBIED
 } call_state;
 
-typedef struct request_matcher request_matcher;
+typedef struct request_matcher_t request_matcher_t;
```

Comment:
Same here.

Reply:
Here too.

Reply:
Let's rename the fields on lines 148 and 174 from `request_matcher` to `matcher`.

Reply:
done